notebooks/book1/10/logreg_poly_demo.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_plot()
</pre><p>Can we get rid out of this warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>